src/lib.rs

```diff
@@ -183,7 +183,7 @@ fn write_polygon<T: NumTy>(buffer: &mut Vec<u8>, p: &Polygon<T>) {
     let num_rings = (interiors.len() + 1) as u32;
     write_u32(buffer, num_rings);
     write_ring(buffer, exterior);
-
+    // is there somewhere in the spec that says that the exterior is always the first ring? Or is that an arbitrary choice?
```

Comment:
I can't find it in any specs but it's typically the case that the first ring is the exterior.